EIPS/eip-5388.md

```diff
@@ -0,0 +1,151 @@
+---
+eip: 5388
+title: Token-Gated HTTPS Endpoints
+description: Composable RESTful and Solidity interface to implement token-gated HTTPS endpoints usind data tokens.
+author: Tim Daubenschütz (@TimDaub)
+discussions-to: https://ethereum-magicians.org/t/token-gated-https-endpoints/10205
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-08-01
+requires: 712
```

Comment:
```suggestion
requires: 20, 712
```

Reply:
I'm against adding EIP-20 as a requirement. The first sections use EIP-20 as an example for payments. But essentially, this document isn't opinionated which smart contract method is being called or if tokens are transferred as a result of it.